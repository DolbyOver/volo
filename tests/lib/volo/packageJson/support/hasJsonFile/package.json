{
    "name": "hasJsonFile",
    "version": "2.0"
}
