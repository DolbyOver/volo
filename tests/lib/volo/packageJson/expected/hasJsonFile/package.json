{
  "name": "hasJsonFile",
  "version": "2.0",
  "volo": {
    "dependencies": {
      "dijit": "dojo/dijit/1.7.2"
    }
  }
}